tests/swagger_generation/test_swagger_generator.py

```diff
@@ -146,6 +146,7 @@ class Error(m.Schema):
                 },
             )
         ],
+        include_hidden=False,
```

Comment:
include_hidden in the SwaggerV3Generator defaults to false, so you can leave this out and not modify this file.

Reply:
good call ! this fall should not be modified anyway since I made a separate test file